cmake failure to find netcdf-cxx

When building standalone cmake fails as follows:

CMake Error at /home/users/david.davies/cylc-run/mi-bg854/share/installs/ioda-spicesystem/lib64/cmake/ioda/ioda-targets.cmake:105 (set_target_properties):
  The link interface of target "ioda" contains:

    NetCDF::NetCDF_CXX

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /home/users/david.davies/cylc-run/mi-bg854/share/installs/ioda-spicesystem/lib64/cmake/ioda/ioda-config.cmake:74 (include)
  /home/users/david.davies/cylc-run/mi-bg854/share/installs/cmake-spicesystem/share/cmake-3.31/Modules/CMakeFindDependencyMacro.cmake:76 (find_package)
  /home/users/david.davies/cylc-run/mi-bg854/share/installs/ufo-spicesystem/lib64/cmake/ufo/ufo-import.cmake:38 (find_dependency)
  /home/users/david.davies/cylc-run/mi-bg854/share/installs/ufo-spicesystem/lib64/cmake/ufo/ufo-config.cmake:64 (include)
  CMakeLists.txt:61 (find_package)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -47,7 +47,7 @@ find_package( MPI REQUIRED COMPONENTS C CXX Fortran )
 include_directories( ${Boost_INCLUDE_DIR} )
 
 # NetCDF
-find_package( NetCDF REQUIRED COMPONENTS C Fortran)
+find_package( NetCDF REQUIRED COMPONENTS C CXX Fortran)
 
 # eckit
 find_package( eckit REQUIRED )

